core/overdrive.py

```diff
@@ -1024,24 +1024,47 @@ def book_info_to_circulation(self, book):
             patrons_in_hold_queue = 0
         elif book.get("isOwnedByCollections") is not False:
             # We own this book.
-            for account in book.get("accounts", []):
-                # Only keep track of copies owned by the collection
-                # we're tracking.
-                if account.get("id") != self.library_id:
-                    continue
+            accounts = book.get("accounts", [])
+            # consortial accounts will always contain an account with id == -1
+            # This section of code is convoluted in part because I don't fully
+            # understand the relationship between overdrive, overdrive
+            # advantage, and overdrive consortial accounts and how related
+            # information is returned by the API or not depending on the
+            # the API token used.
+            # The problem as I see it is that the response is overloaded:
+            # you need to know something about the nature of the API token
+            # in order to properly interpret the results...maybe.  I'm not sure
+            # talking to OD about it has not left me with clarity on the issue.
+            is_consortium = len([a for a in accounts if a["id"] == -1]) == 1
+            licenses_owned = 0
+            licenses_available = 0
+            if self.library_id != -1:
+                # this is an overdrive advantage account
+                for account in accounts:
+                    if account["id"] == self.library_id:
+                        if "copiesOwned" in account:
+                            licenses_owned += int(account["copiesOwned"])
+                        if "copiesAvailable" in account:
+                            licenses_available += int(account["copiesAvailable"])
+            elif not is_consortium:
+                # this is a non-consortial overdrive account
+                # just get the values from the book object
```

Comment:
Minor/preference - style/readability: use full sentences, standard capitalization, and punctuation.